monitor/src/main/java/edu/asu/diging/monitor/web/HomeController.java

```diff
@@ -18,4 +24,10 @@ public String home(Model model) {
     		model.addAttribute("apps", appManager.getApps());
         return "home";
     }
+    
+    
+    @RequestMapping(value="/admin/apps/reload",method= RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
since you put this at `/admin/apps/reload` it won't work if the user is not logged in. It should though.